ddl/serial_test.go

```diff
@@ -613,6 +613,23 @@ func (s *testSerialSuite) TestCreateTableWithLikeAtTemporaryMode(c *C) {
 	c.Assert(err, IsNil)
 	tableInfo := table.Meta()
 	c.Assert(len(tableInfo.ForeignKeys), Equals, 0)
+
+	// Issue 25613.
+	tk.MustExec("drop table if exists tb2, tb3")
+	tk.MustExec("create table tb2(id int);")
+	tk.MustExec("create global temporary table tb3 like tb2 on commit delete rows;")
```

Comment:
Try `create global temporary table tb3 like tb2` and `create table tb3 like tb2 on commit delete rows` first.